in_toto/exceptions.py

```diff
@@ -1,4 +1,4 @@
-from in_toto.ssl_commons.exceptions import Error
+from securesystemslib.exceptions import Error
```

Comment:
```Python
class RuleVerficationFailed(Error):
```
The exception name here is inconsistent with the others.  Should it instead be `RuleVerificationFailedError`?